<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="rOrdinaria">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordinaria</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rPlazoFijo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Plazo Fijo</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rEntidad">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Entidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione el tipo de cuenta a cobrar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Saldo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="direcPeriodo">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Periodo Pago</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>320</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingreso Frecuente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelState">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineID">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinIngreso">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rExtraer">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Extraer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rDepositar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>107</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Depositar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Operaciones</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinAmmount">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Solicitar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QLabel" name="lista1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>211</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QLabel" name="list2BySaldo">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>211</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string>Lista</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QLabel" name="mayorSaldo">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mayor Saldo</string>
    </property>
   </widget>
   <widget class="QLabel" name="historylabel">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>460</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Historial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>420</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Solicitar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Obtener</string>
    </property>
   </widget>
   <widget class="QLabel" name="tiposCuenta">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipos de Cuenta</string>
    </property>
   </widget>
   <widget class="QLabel" name="mayorInteres">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>510</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mayor Interes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>450</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="editSaldoF">
    <property name="geometry">
     <rect>
      <x>761</x>
      <y>40</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
